tutorial/Lab4/README.md

```diff
@@ -32,54 +30,45 @@ In this part of the lab, we show you how to install the `guestbook` chart from t
    
    ```"helmm101" has been added to your repositories```
```

Comment:
can you please fix this too from helmm101 to helm101 :) thanks

Reply:
Good catch, fixed